# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vlock/disable-pam.diff
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vlock-1.3/Makefile.vanilla	2007-06-18 19:48:38.531779497 +0200
+++ vlock-1.3/Makefile	2007-06-18 19:48:58.584922260 +0200
@@ -3,8 +3,8 @@
 CC = gcc
 # remove the -DUSE_PAM, -ldl, and -lpam if you aren't using PAM
 RPM_OPT_FLAGS=-O2
-CFLAGS = $(RPM_OPT_FLAGS) -DUSE_PAM
-LIBS = -ldl -lpam -lpam_misc
+CFLAGS = $(RPM_OPT_FLAGS)
+LIBS = -lcrypt
 
 OBJS = vlock.o signals.o help.o terminal.o input.o
 
